src/docs/patterns/tables.mdx

```diff
@@ -0,0 +1,463 @@
+        `Row actions can be placed at the end of the row.`,
+        `DO: On desktop, you can choose to have row actions visible only when the mouse is hovering on the row. The buttons won't compete for rooms with the rest of the table.`,
+    ]}
+/>
+
+<ImageGrid
+    images={[Drag1, Drag2]}
+    caption={[
+        `Drag-and-drop actions are usually placed at the beginning of a row. If you have row numbers, you may choose to display the drag handle on hover.`,
+    ]}
+/>
+
+## Batch Actions
+
+If your application allows users to perform batch actions to their data set, such as deleting 15 items at once, you can add a checkbox column on the left rim of the table.
+
+<ImageGrid
+    images={[BatchAction1, BatchAction2]}
+    captionsUnderImages={[
+        `Buttons are disabled when no rows are checked.`,
+        `Buttons become enabled when rows are checked. Optionally, use theme color to highlight the selected row, and label the button with the number of items selected to account for pagination.`,
+    ]}
+/>
+
+<!--
+
+Note: I think this is a good suggestion, but might be too high of a requirement for our adopters
+
+You may change the clickable area when users are cognitively loaded and busy deciding which row to check. When they enter this "batch action mode", increase the row selection area so that it is easier to select.
+
+<ImageGrid
+    images={[ClickArea1, ClickArea2]}
+    captionsUnderImages={[
+        `When nothing is selected, the selection target area is limited to the region around the checkbox.`,
+        `When a row is selected, the application enters this "batch action mode", and users can select other rows by clicking anywhere on those rows.`,
+    ]}
+/>
+-->
+
+## Sticky Header Row
+
+When a table gets long, your users should not need to scroll back to the top to view the table header.
+
+<ImageGrid
+    images={[StickyHeader]}
+    caption={[`Use sticky headers to help your users navigate through the table.`]}
+    regularWidth
+/>
+
+## Expanded Row
+
+Occasionally, table rows may expand to allow users _take a quick peek_ into a small amount of row details.
+
+<ImageGrid
+    images={[ExpandedRow]}
+    caption={[`Users can quickly glance into a small amount of row details.`]}
+    regularWidth
+/>
+
+<ImageGrid
+    images={[ExpandedRowTooComplicated, ExpandedRowActions]}
+    captionsUnderImages={[
+        `DONT: Do not include complicated contents in an expanded row, as this will break the efficiency of a table. In this case, you should design a dedicated page for the content.`,
+        `DONT: Do not surprise users by hiding high priority actions in the expanded row space.`,
+    ]}
+/>
+
+## Horizontal Scrolling
+
+When your table overflows horizontally, you should allow your users to scroll horizontally to view more about the table. However, this interaction is less ergonomically convenient when users are not using a touch screen or a trackpad. See the [Responsive](#responsive) section below.
```

Comment:
I think we should be discouraging horizontal scrolling and encouraging people to only show as much data as is necessary.

Reply:
I added a `CAUTION:` tag to the section